postgres_schema.sql

```diff
@@ -2,12 +2,11 @@
 DROP TABLE IF EXISTS reviews;
 
 CREATE TABLE reviews (
-  id SERIAL,
+  id SERIAL PRIMARY KEY,
   product_id INT NOT NULL,
-  review_id INT NOT NULL PRIMARY KEY,
   rating INT NOT NULL,
   date BIGINT NOT NULL,
-  summary VARCHAR(100) NOT NULL,
+  summary VARCHAR(1000) NOT NULL,
```

Comment:
interesting, because this is supposed to max out at 60, can use VARCHAR or TEXT in Postgres if length is unknown